low-level/files/opt/seagate/sspl/setup/sspl_reset.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+
+from cortx.sspl.bin.sspl_constants import (PRODUCT_FAMILY,
+ file_store_config_path, enabled_products)
+from cortx.utils.process import SimpleProcess
+from cortx.utils.service import Service
+
+SSPL_CONFIGURED=f"/var/{PRODUCT_FAMILY}/sspl/sspl-configured"
+
+class SSPLResetError(Exception):
+    """ Generic Exception with error code and output """
+
+    def __init__(self, rc, message, *args):
+        self._rc = rc
+        self._desc = message % (args)
+
+    def __str__(self):
+        if self._rc == 0: return self._desc
+        return "error(%d): %s" %(self._rc, self._desc)
+
+
+class SSPLResetCmd:
+
+    def __init__(self, args: list):
+        self.args = args
+        self.name = "sspl_reset"
+        self._config()
+
+    def _config(self):
+        if os.path.exists(file_store_config_path):
+            with open(file_store_config_path, "r") as SSPL_CONF:
+                for line in SSPL_CONF:
+                    if 'data_path' in line:
+                        self.SSPL_DATA_DIR=line.split("=")[1]
+                    elif 'log_file_path' in line:
+                        self.IEM_FILE_PATH=line.split("=")[1]
+        else:
+            self.SSPL_DATA_DIR=f"/var/{PRODUCT_FAMILY}/sspl/data"
+            self.IEM_FILE_PATH=f"/var/{PRODUCT_FAMILY}/sspl/data/iem/iem_messages"
+    
+    def process(self):
+        if not self.args:
+            raise SSPLResetError(errno.EINVAL, "Invalid Argument for %s"% self.name)
+        i=0
+        while i < len(self.args):
+            try:
+                if self.args[i] == "hard":
+                    HardCmd(self.args).process()
+                elif self.args[i] == "soft":
+                    SoftCmd(self.args).process()
+            except (IndexError, ValueError):
+                raise SSPLResetError(errno.EINVAL, "Invalid Argument for %s"% self.name)
+            i+=1
+
+    def del_file(self, filename):
+        if os.path.exists(filename):
+            os.remove(filename)
+
+
+class HardCmd(SSPLResetCmd):
+    """Hard reset Cmd"""
+    name = "hard"
+    user_present=False
+    
+    def __init__(self, args):
+        super().__init__(args)
+
+    def process(self):
+        i = 0
+        while i < len(self.args):
+            try:
+                if self.args[i] == '-p':
```

Comment:
-p kind of parsing would go away as conf stor will have full keys to validate with. @mariyappanp to take care

Reply:
Yes. Removed this block of code and validated it in sspl_setup script.